src/web/source.rs

```diff
@@ -29,7 +29,7 @@ struct File {
 }
 
 /// A list of source files
-struct FileList {
+pub(crate) struct FileList {
```

Comment:
Why did this change?

Reply:
That actually gets deleted later on, it was supposed to get removed in a later stage but I guess it slipped through

Reply:
Ok, can you change it back?